### DISCUSSION TITLE
Red Notebook crashes on Raspberry Pi Os 64 bit

### DISCUSSION THREAD

**Discussion by ebs54** [jendrikseipp/rednotebook#740] [Q&A]
Hello, I installed Red Notebook from the repo on the newest 64 bit Raspberry pi os. It starts but after 1 second the screen in the Red Notebook windows crashes and show only snow. Is this fail known?

**Comment by jendrikseipp** [comment]
Thanks for the report! Is there any output to the terminal?

**Comment by ebs54** [comment]
Thanks for your fast feed back.There is nothing but snow to see.Auf Deutsch kann ich besser,Eberhard KühneGroßdubrauvom Handy
-------- Ursprüngliche Nachricht --------Von: Jendrik Seipp ***@***.***> Datum: 02.01.24  10:27  (GMT+01:00) An: jendrikseipp/rednotebook ***@***.***> Cc: ebs54 ***@***.***>, Author ***@***.***> Betreff: Re: [jendrikseipp/rednotebook] Red Notebook crashes on Raspberry Pi
  Os 64 bit (Discussion #740) 
Thanks for the report! Is there any output to the terminal?
